mission/config/functions.hpp

```diff
@@ -220,7 +220,7 @@ class CfgFunctions
 
 		class system_dac_cong {
 			file = "functions\systems\dac_cong";
-
+			class daccong_respawns_delete_all {};
```

Comment:
i've not changed `vn_mf_fnc_director_close_zone` to use this new function yet as that'll cause a merge conflict... and i'm being lazy. (I should just change it) see #360